app/assets/stylesheets/foundation_and_overrides.scss

```diff
@@ -1403,16 +1405,16 @@ $include-html-global-classes: $include-html-classes;
 // $include-html-top-bar-classes: $include-html-classes;
 
 // Background color for the top bar
-// $topbar-bg-color: $oil;
+   $topbar-bg-color: rgba(0,0,0,1);
```

Comment:
Line should be indented 0 spaces, but was indented 3 spaces<br>Commas in function arguments should be followed by a single space